app/controllers/developers_controller.rb

```diff
@@ -46,6 +46,10 @@ def show
 
   private
 
+  def developers
+    Developer.looking_for_work.most_recently_added.with_attached_avatar
```

Comment:
As this is getting duplicated and this is probably going to be used in multiple places, adding a class method on the Developer model is not a bad idea

Reply:
Once we use it three times we can extract something. But for now I like how explicit the query is.